src/engine/Loader.ts

```diff
@@ -258,7 +259,8 @@ export class Loader extends Class implements Loadable<Loadable<any>[]> {
           this._playButton.click();
         }
       });
-      const promise = new Promise<void>((resolve) => {
+      this._positionPlayButton();
```

Comment:
Small bug discovered, the button isn't positioned until the first update, this makes sure it's in the right position immediately